hrms/hr/report/shift_attendance/shift_attendance.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2023, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+from frappe.utils import flt
+
+
+def execute(filters=None):
+	columns = get_columns()
+	data = get_data(filters)
+	return columns, data
+
+
+def get_columns():
+	return [
+		{
+			"label": _("Employee"),
+			"fieldname": "employee",
+			"fieldtype": "Link",
+			"options": "Employee",
+			"width": 120,
+		},
+		{
+			"label": _("Employee Name"),
+			"fieldname": "employee_name",
+			"fieldtype": "Data",
+			"width": 150,
+		},
+		{
+			"label": _("Shift"),
+			"fieldname": "shift",
+			"fieldtype": "Link",
+			"options": "Shift Type",
+			"width": 120,
+		},
+		{
+			"label": _("Attendance Date"),
+			"fieldname": "attendance_date",
+			"fieldtype": "Date",
+			"width": 120,
+		},
```

Comment:
Add column for attendance status

Reply:
Should this be color coded?

Reply:
IG it's fine if we don't. 3 adjacent color-coded columns won't look good probably.